### PR TITLE
Python wrapping update

### PR DESCRIPTION
**Pull request by gagelarsen**

### CODE REVIEW

File: _package/xmscore/misc/observer.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from .._xmscore.misc import Observer as Obs
+
+
+class Observer(Obs):
+    """Mock Observer class for testing."""
+
+    def __init__(self, instance=None):
```
Comment (swainn) [inline _package/xmscore/misc/observer.py]: If not using instance, remove the argument.

File: _package/xmscore/misc/observer.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+from .._xmscore.misc import Observer as Obs
+
+
+class Observer(Obs):
+    """Mock Observer class for testing."""
```
Comment (swainn) [inline _package/xmscore/misc/observer.py]: Add doc strings for python interface.

File: xmscore/python/xmscore_py.cpp [hunk 3]
```diff
@@ -20,16 +20,10 @@ namespace py = pybind11;
...
     m.attr("__version__") = XMS_VERSION;
 
-    const char* misc_doc = R"pydoc(
-        The misc module of the xmscore python library contains classes and
-        functions that are shared between all of the xms family of libraries.
-        These functions and classes can be used in any of the library to ensure
-        that functionality is standardized.
-    )pydoc";
-    py::module modMisc = m.def_submodule("misc", misc_doc);
+    py::module modMisc = m.def_submodule("misc");
```
Comment (swainn) [inline xmscore/python/xmscore_py.cpp]: Where do we move these module level doc strings?

File: _package/xmscore/misc/observer.py (hunk 1 above)
Comment (gagelarsen) [reply _package/xmscore/misc/observer.py]: fixed in commit f6fbbec

File: _package/xmscore/misc/observer.py (hunk 2 above)
Comment (gagelarsen) [reply _package/xmscore/misc/observer.py]: fixed in commit 041b40a

File: xmscore/python/xmscore_py.cpp (hunk 3 above)
Comment (gagelarsen) [reply xmscore/python/xmscore_py.cpp]: This documentation isn't shown, we no longer need it

**Pull request merged**